rc-events-service/src/main/scala/com/epam/random_coffee/events/model/Event.scala

```diff
@@ -0,0 +1,16 @@
+package com.epam.random_coffee.events.model
+
+import akka.http.scaladsl.server.{ PathMatcher, PathMatcher1 }
+import akka.http.scaladsl.server.PathMatchers.Segment
+
+final case class Event(id: EventId, eventName: String)
```

Comment:
```suggestion
final case class Event(id: EventId, name: String)
```
[minor] since you're using just `name` in other places

Reply:
fixed